app/assets/javascripts/admin/orders/controllers/bulk_invoice_controller.js.coffee

```diff
@@ -8,7 +8,7 @@ angular.module("admin.orders").controller "bulkInvoiceCtrl", ($scope, $http, $ti
     $scope.poll_wait = 5 # 5 Seconds between each check
     $scope.poll_retries = 80 # Maximum checks before stopping
 
-    $http.post('/admin/orders/invoices', {order_ids: $scope.selected_orders}).success (data) ->
+    $http.post('/admin/orders/invoices', {order_ids: $scope.selected_orders, q: {s: 'completed_at DESC'}}).success (data) ->
```

Comment:
My big issue is that I don't know how to get easily the list order at this step. Lacking some Angular basics.

Reply:
I think I would add this in the pdf service instead (see below).